src/isaw.policy/src/isaw/policy/profiles/default/memberdata_properties.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0"?>
 <object name="portal_memberdata" meta_type="PlonePAS MemberData Tool">
   <property name="CVReference" type="string"></property>
+  <property name="ProfileReference" type="string"></property>
```

Comment:
You should add an upgrade step to actually apply this change to the portal_memberdata tool.